Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -1,270 +1,249 @@
 ﻿using System.Text.RegularExpressions;
 
-namespace Frends.SFTP.DownloadFiles.Definitions
+namespace Frends.SFTP.DownloadFiles.Definitions;
+
+///<summary>
+/// Policies for creating names for remote files: expands macros etc.
+///</summary>
+internal class RenamingPolicy
 {
-    ///<summary>
-    /// Policies for creating names for remote files: expands macros etc.
-    ///</summary>
-    internal class RenamingPolicy
+    private IDictionary<string, Func<string, string>> MacroHandlers;
+    private IDictionary<string, Func<string, string>> SourceFileNameMacroHandlers;
+
+    public RenamingPolicy() : this("", Guid.Empty)
```

Comment:
You are not using empty constructor, so it can be removed.

Reply:
Removed.